client/src/LayoutComponents/StudentProfile/StudentProfile.js

```diff
@@ -30,11 +33,14 @@ class StudentProfile extends Component {
   render() {
     const { studentInfo, studentProjects } = this.state;
     const projects = studentProjects.map((project, i) => {
-      return(
-        <Link to={'/project/' + project.project_id} key={i}>
-         <Project key={project.project_id} projectImg={project.project_image} />
-         </Link>
-)
+      return (
+        <Link to={`/project/${project.project_id}`} key={i}>
```

Comment:
@Rahaf-96 You could use the project_id for the key instead of i, which would be a little more specific.  React uses key to identify which elements it can reuse or which it has to discard when an array / map re-renders.